CONTRIBUTING.md

```diff
@@ -217,6 +218,22 @@ There are many algorithms to solve this problem. You may implement any of them.
 [great document by Jeff Erickson](http://jeffe.cs.illinois.edu/teaching/compgeom/notes/01-convexhull.pdf)
 for more details about the problem and the different algorithms to solve it.
 
+### Factorial rules
+
+The factorial of an integer `n` is defined as:
+
+`n! = 1 x 2 x 3 x 4 x ... x n`
+
+With the special case `0! = 1`.
+
+You must write a program that takes an integer `n` as input, and outputs `n!`.
```

Comment:
Looks great! I might even be more specific here and specify that the program should be executable. In other words, the program should accept `n` on standard input via the command line and output `n!` to standard output.